ui/src/router/components/form/components/DefaultAutoscalingPolicyPanel.js

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import { Panel } from "./Panel";
+import {
+  EuiFieldText,
+  EuiForm,
+  EuiFormRow,
+  EuiSpacer,
+  EuiSwitch,
+} from "@elastic/eui";
+import { FormLabelWithToolTip } from "../../../../components/form/label_with_tooltip/FormLabelWithToolTip";
+import { useOnChangeHandler } from "../../../../components/form/hooks/useOnChangeHandler";
+import { DescribedFormGroup } from "@gojek/mlp-ui";
+
+export const DefaultAutoscalingPolicyPanel = ({
+  useDefaultAutoscalingPolicy,
+  toggleUseDefaultAutoscalingPolicy,
+  autoscalingPolicyConfig,
+  onChangeHandler,
+  errors = {},
+}) => {
+  const { onChange } = useOnChangeHandler(onChangeHandler);
+
+  return (
+    <Panel title="Resources Requirements">
+      <EuiForm>
+        <EuiSpacer size={"m"}/>
+
+        <EuiSwitch
+          label={useDefaultAutoscalingPolicy ?
+            "Use default autoscaling policy"
+            : "Customise resources requirements and autoscaling manually"}
+          checked={useDefaultAutoscalingPolicy}
+          onChange={toggleUseDefaultAutoscalingPolicy}
+        />
+
+        {useDefaultAutoscalingPolicy &&
+          <>
+            <EuiSpacer size={"m"}/>
+
+            <DescribedFormGroup description={
+              "The value specified here will be used to perform autoscaling automatically. There is no need to specify " +
+              "and resource requests values or custom autoscaling values."
```

Comment:
I think the `and` here should be `any`, no?

Reply:
Thanks for catching this! 😅